zendserver/extensions.sls

```diff
@@ -11,6 +11,10 @@
 # Get the key
 {% set zend_api_key = salt['grains.get']('zendserver:api:key') %}
 
+# Get JQD settings
+{% set jqd_http_retries = salt['pillar.get']('zendserver:jqd:http_retry_count', '0') %}
```

Comment:
Same for this one, lets default it to 10

Reply:
Resolved by c40f4fa058475dede96d221c0c0bebd1af148647